src/main/java/picard/arrays/GtcToVcf.java

```diff
@@ -538,6 +541,9 @@ private VCFHeader createVCFHeader(final Build37ExtendedIlluminaManifest manifest
         if (EXPECTED_GENDER != null) {
             lines.add(new VCFHeaderLine(InfiniumVcfFields.EXPECTED_GENDER, EXPECTED_GENDER));
         }
+        if (PIPELINE_VERSION != null) {
+            lines.add(new VCFHeaderLine(InfiniumVcfFields.PIPELINE_VERSION, PIPELINE_VERSION.toString()));
```

Comment:
Since `PIPELINE_VERSION` is already a String you don’t need to call `toString`
```suggestion
            lines.add(new VCFHeaderLine(InfiniumVcfFields.PIPELINE_VERSION, PIPELINE_VERSION);
```